src/helix-ui/styles/components.less

```diff
@@ -32,7 +32,11 @@
 @import 'components/stepper'; // BETA
 @import 'components/tables';
 @import 'components/tabs';
-@import 'components/text-inputs';
+@import 'components/text-input';
+@import 'components/textarea';
 @import 'components/toasts';
 @import 'components/tooltips';
 @import 'components/typography';
+
+// DEPRECATED
+@import 'components/hxTextCtrl'; // remove in v1.0.0
```

Comment:
`components/text-inputs` was renamed to `components/hxTextCtrl` to avoid confusion with `components/text-input` (singular)